### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Moathal**
### Pop-up window to show projects.
- Added the markup HTML for the pop up modal.
- Added styling for the pop up modal.
- Implemented a JavaScript function to render information from an array of objects for each project to show on pop up modal.
- Added a JavaScript to open and close pop up window.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,405 +1,347 @@
...
-                  See project
-                </button>
-              </div>
+    <div id="portfolio" class="sub-container-2 flex column">
+      <section class="recentWork flex column">
+        <div class="hdr flex column">
+          <h2 class="mainTxt-2">MY RECENT WORK</h2>
+          <hr class="divider" />
+        </div>
+        <div class="workBlocks">
```
Comment (topeogunleye) [inline index.html:163]: - [ ] Great job adding your project modals dynamically, however you also need to add your project cards themselves dynamically as you did for the project modal.
- [ ] I kindly suggest that you remove the project cards you added manually in your HTML and add them dynamically through JavaScript as you did for the modal

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @Moathal,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Moathal) [reply index.html:163]: This was week2 assignment, and I don't really have any projects to show yet. However the project requirement is to do the pop up modal only as it is mentioned in the title.

**Review by ishimwezachee — APPROVED** [review]
Hi @Moathal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
Project is running successflly 👍 
PR is well described 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**